pages/about.html

```diff
@@ -4,9 +4,68 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="styles/about.css" rel="stylesheet">
+  <title>About Me</title>
 </head>
-<body>
-  
+<body class="container">
+
+  <header class="header">
+    <h1>About Me</h1>
+  </header>
+
+  <nav class="nav">
+    <a href="index.html">Home</a>
+    <a href="portfolio.html">Programming Portfolio</a>
+  </nav>
+
+  <p>
+    <h1>Background</h1>
+    I am from Socorro, TX. Usually say I am from El Paso because more people know the city and we are a part of 
+    the same county. B.S. in Geology. Worked a few different fields before persuing software development. I am currently located in Chicago, IL with my husband, Aaron Ortega, and our 6yr old germand shepherd husky, Soos
+    <ul>
+      <h3>Where I have been:</h3>
+      <li>substitute teacher</li>
+      <li>cake decorator</li>
+      <li>barista</li>
+      <li>bubble tea maker!</li>
+      <li>lab tech for undergraduate chemistry labs</li>
+    </ul>
+
+    <ul>
+      <h3>Where I am going:</h3>
+      <li>Software Developer!!! ＼(＾O＾)／</li>
+    </ul>
+  </p>
+
+  <p>
+    <h2>Interests</h2>
+    Geology, tennis, crafting, travel, baking and cake decorating. 
+    I am subscribed to way too many youtube channels. Once I find a show or movie I like, I will rewatch it and love it even more after every watch cycle. 
+    <ul>
+      <h4>Some stuff I can watch on repeat:</h4>
+      <li>Rodgers and Hammerstein's Cinderella (The one with Brandy, Whitney Houston, Bernadette Peters ...)</li>
+      <li>Romance is a Bonus Book</li>
+      <li>Aggretsuko</li>
+      <li>One Punch Man</li>
+      <li>Golden Girls</li>
+      <li>Bob's Burgers</li>
+      <li>What We do in the Shadows</li>
+      <li>RuPaul's Drag Race (excluding season 7 ... but I love Katya and Trixie)</li>
+      <li>Jersey Show (trash but I love it!)</li>
+      <li>sooooo much more ... </li>
+    </ul>
+  </p>
+
+  <section class="section">
```

Comment:
A section tag _does_ want a child h tag to provide a heading for the section. It seems like this is mostly here to provide a flex container, so feel free to use non-semantic tags when they are being introduced for styling purposes.